### ISSUE TITLE
Add support for clickable cells/rows in Table Visualization

### ISSUE THREAD

**Issue by sharmaashish** [sharmalab/Datascope#50]
Users may want to support the launch of a new entity using elements of a row or a cell. We should support this.

**Comment by lastlegion** [comment]
This also solves the issues raised in #49. 

I think we need to extend `visualization.json` to support this links. Currently visualization.json looks like:

```
{
    "visualizationType": "dataTable",
    "attributes":[
    	{"attributeName": "id"},
    	{"attributeName": "Ai"},
    	{"attributeName": "Di"}
    ]
}
```
## For row clickable

```
{
    "visualizationType": "dataTable",
    "attributes":[
    	{"attributeName": "id"},
    	{"attributeName": "Ai"},
    	{"attributeName": "Di", "rowClickable": true}
    ]
}
```

## For cell

```
{
    "visualizationType": "dataTable",
    "attributes":[
    	{"attributeName": "id"},
    	{"attributeName": "Ai" , "cellClickable": true},
    	{"attributeName": "Di", "cellClickable": true}
    ]
}
```

we can discuss more about the spec.

**Comment by birm** [comment]
solved with a01b5e31f34853683840cc7453bb88f852e58668
